### PR TITLE
Add ribocutter

### PR DESCRIPTION
**Pull request by Delayed-Gitification**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by Delayed-Gitification** [comment]
@BiocondaBot please add label

File: recipes/ribocutter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  imports:
+    - ribocutter
+  commands:
+    - ribocutter --help
+
+about:
+  home: "https://github.com/Delayed-Gitification/ribocutter.git"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (druvus) [inline recipes/ribocutter/meta.yaml:40]: Please include the license file.

File: recipes/ribocutter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - ribocutter=ribocutter.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```
Comment (druvus) [inline recipes/ribocutter/meta.yaml:20]: You only need to include python and pip in the host section.

File: recipes/ribocutter/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+
+test:
+  imports:
+    - ribocutter
+  commands:
+    - ribocutter --help
+
+about:
+  home: "https://github.com/Delayed-Gitification/ribocutter.git"
```
Comment (druvus) [inline recipes/ribocutter/meta.yaml:37]: This is not a valid url

File: recipes/ribocutter/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "Design oligos to produce sgRNAs for abundant sequences in a fastq file"
+  doc_url:
```
Comment (druvus) [inline recipes/ribocutter/meta.yaml:42]: Remove doc_url and dev_url if it is not used.

**Review by druvus — CHANGES_REQUESTED** [review]

File: recipes/ribocutter/meta.yaml (hunk 1 above)
Comment (Delayed-Gitification) [reply recipes/ribocutter/meta.yaml]: is it ok if I just removed this line? I have done so in the latest version

**Review by druvus — APPROVED** [review]

**Comment by druvus** [comment]
@BiocondaBot please merge

**Pull request merged**